core/src/main/res/values/colors.xml

```diff
@@ -38,4 +38,10 @@
     <color name="dialog_filter_clear_inactive_dark">#bbbbbb</color>
     <color name="dialog_filter_inactive_light">#eeeeee</color>
     <color name="dialog_filter_inactive_dark">#555555</color>
+
+    <!-- swipe actions -->
+    <color name="swipe_light_green_200">#c5e1a5</color>
+    <color name="swipe_light_red_200">#ef9a9a</color>
+    <color name="swipe_light_blue_200">#81d4fa</color>
+    <color name="swipe_yellow_200">#fff59d</color>
```

Comment:
How about using colors with transparency, so they are not as bright in the dark theme?

Reply:
These are just placeholders I wasn't paying much attention to design/colors, I hoped @keunes could have a look at this? :)

Reply:
> How about using colors with transparency, so they are not as bright in the dark theme?

That could be a nice idea. That way, also just one colour needs to be defined per action. However, I'm not sure if that'll work nicely (e.g. a yellow with transparency on a dark gray might become a 'dirty' colour). I'll define the exact colours (and see if working with transparency works) later.

I think we need colours for the following actions (I list the colours without having checked the PR):
* Add to queue -> green-ish
* Download -> blue-ish
* ~~Remove 'new' flag~~ [action should be depricated, for several reasons, so I won't think about a colour for now]
* Add to Favorites -> yellow-gold-ish
* Mark as played -> grey-ish
* Remove from queue -> dark-blue-ish

And I [proposed](https://design.penpot.app/#/view/e357a320-7535-11eb-8a88-eb0147e048f5/e357a321-7535-11eb-8a88-eb0147e048f5?token=L4p-9sogmtEVgXkHrtErlg&index=0) the following options:
* Ignore -> red-ish
* Remove from inbox -> dark-blue-ish (can be the same colour as `Remove from queue` as both options won't be available in the same screen, if we want to avoid using the same colour for different actions, we can add purple that I introduced for the brand palette)

Reply:
Remove from new is actually used to remove items from inbox (only new ones), so I think we shoulpd keep it?

Reply:
> Remove from new is actually used to remove items from inbox (only new ones), so I think we shoulpd keep it?

As mentioned in person I have some thoughts about the current states and how they (don't) support the user needs. Will create a separate issue for it.

Reply:
Ok, but thats seperate, lets focus on getting this PR merged and getting basic SwipeActions off the grounds and refining UX later :)

Reply:
Ok, what do you think of these colours?
![Siwipe colours](https://user-images.githubusercontent.com/11229646/124806113-9da8c200-df5c-11eb-9d14-3c3ffe92ab5c.png)

(black = AMOLED theme; grey = dark theme; white = light theme)

Reply:
If ok, I'll type up the colour codes.

Reply:
Actually, I'm thinking now that we wanted swipes to toggle, so that would mean that 'add to queue' and 'remove from queue' would be merged. In that case I'd go for the another blue (in between light & dark).
@ueen @ByteHamster

Reply:
Looks good, did you see the latest demo with the colored icons, maybe these colors could be applied to the icons as well? 
I think merging isn't a good idea, I think that will be confusing when choosing the actions in the dialog, where remove is the default for queue and add to queue will be the default on the inbox, so I think although somewhat redundant, we should remove from queue and add the queue (that is also a toggle) :)